src/Atmos/Parameterizations/CloudPhysics/Microphysics.jl

```diff
@@ -1,271 +1,550 @@
 """
     one-moment bulk Microphysics scheme
 
-Microphysics parameterization based on the ideas of Kessler_1995:
+Included processes
   - condensation/evaporation as relaxation to equilibrium
   - autoconversion
   - accretion
-  - rain evaporation
-  - rain terminal velocity
+  - evaporation and sublimation
+  - terminal velocity
 """
 module Microphysics
 
+using SpecialFunctions
+
 using ClimateMachine.MoistThermodynamics
+
 using CLIMAParameters
 using CLIMAParameters.Planet: ρ_cloud_liq, R_v, grav
-using CLIMAParameters.Atmos.Microphysics:
-    C_drag,
-    MP_n_0,
-    K_therm,
-    τ_cond_evap,
-    τ_sub_resub,
-    τ_acnv,
-    q_liq_threshold,
-    E_col,
-    D_vapor,
-    ν_air,
-    N_Sc,
-    a_vent,
-    b_vent
+using CLIMAParameters.Atmos.Microphysics
 
 const APS = AbstractParameterSet
+const ASuspPS = AbstractSuspendedWaterParameterSet
+const AFallPS = AbstractFallingWaterParameterSet
+const ACPS = AbstractCloudParameterSet
```

Comment:
What's the difference between suspended water and cloud parameter sets?

Reply:
Suspended water is cloud water or cloud ice. Falling water is rain or snow. It helps with dispatching. I can write one accretion function that will accept one argument of type suspended and one argument of type falling. And then depending whether it is cloud or ice, rain or snow it will choose the correct parameters for the collision kernel.

Reply:
Sounds good. It would still seem clearer to me to call is `cloud` and `precip`. Could that be done without overlap with the cloud parameter set defined later (or calling the latter something else)?

Reply:
I'll change to `precip` (rain or snow) and `cloud` (liquid or ice)

Reply:
done